src/frontend/typer/beregning.ts

```diff
@@ -1,4 +1,22 @@
 import { INøkkelPar } from './common';
+import { BehandlingKategori } from './behandling';
+import { IPerson } from './person';
+
+export interface IOppsummeringBeregning {
+    periodeFom: string;
+    periodeTom: string;
+    sakstype: BehandlingKategori;
+    beregningDetaljer: IBeregningDetalj[];
+    stønadstype: YtelseType[];
+    antallBarn: number;
+    utbetaltPerMnd: number;
+}
+
+export interface IBeregningDetalj {
+    person: IPerson;
+    stønadstype: YtelseType;
```

Comment:
I IPersonBeregning nedenfor kalles dette feltet `ytelseType`. Ikke så viktig, men med mindre det er grunn til å differensiere, kanskje vi kan bruke det samme i IOppsummeringBeregning, IBeregningDetalj og IPersonBeregning? :)

Reply:
Ja, jeg kan endre det :)

Reply:
@marenger @charliemidtlyng greit for dere?

Reply:
Kan godt endre til `ytelseType` / `ytelsestype` for min del. Det burde vi kanskje ha gjort tidligere - da jeg ikke skjønte at stønadstype skulle mappes til ytelsestype da vi satt og kodet 😂

Reply:
Husk å endre det backend hvis du endrer det her @henninghaakonsen.

Reply:
👍